package.json

```diff
@@ -41,7 +41,7 @@
   "name": "restify",
   "homepage": "http://restifyjs.com",
   "description": "REST framework",
-  "version": "4.3.3",
+  "version": "4.3.4",
```

Comment:
Please remove the version, we use a semi-automated publishing process which will handle it.

Reply:
Fixed in https://github.com/restify/node-restify/pull/1656/commits/dd8f3dd8533abc6bdcfcb79e03100ed0bfb04c47